src/components/SearchInput.vue

```diff
@@ -56,7 +57,7 @@ export default {
      * @param {Number} index - the index in the `suggestions` array of the item selected.
      */
     selectItem(index) {
-      this.selections.push(this.suggestions[index]);
+      this.$emit('itemSelected', this.suggestions[index]);
```

Comment:
Had to look this up. So this pushes up to `QuestionsPage`?

Reply:
Exactly. This sends a custom event that `QuestionsPage` listens for and uses to capture the selected term. It's basically equivalent to an Ember action.